website_animate/static/src/js/o_animate.editor.js

```diff
@@ -34,7 +34,7 @@ sOptions.registry.o_animate = sOptions.Class.extend({
                         'visibility': '',
                     });
         if (this.$target.hasClass('o_animate')) {
-            this.$target.css('animation-play-state', 'paused');
```

Comment:
I dont know why this line was usefull.. this property is anyway settled with 'paused' in the start method of o_animate.frontend.js

Reply:
There may be no reason at all, that's why we need to review the entire app in master :wink: